revocation.ipldsch

```diff
@@ -0,0 +1,10 @@
+type RevocationAction <: Action {
+  cmd "ucan/revoke"
+  nnc ""
+  arg RevocationArguments
+}
+
+type RevocationArguments struct {
+  rev &Delegation
+  pth [&Delegation]
+}
```

Comment:
Even more things that I want from IPLD Schema:
- Subtyping
- Concrete values

I'd even be happy extending...

> `nullable` for map values, list values, and struct fields
> `optional` for struct fields
> `implicit` for struct fields -- in some representations

...with something like...

> `exact` for struct _values_